easybuild/easyconfigs/r/Reapr/Reapr-1.0.18-foss-2019a-Perl-5.28.1.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'MakeCp'
+
+name = 'Reapr'
+version = '1.0.18'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://www.sanger.ac.uk/science/tools/reapr'
+description = """A tool that evaluates the accuracy of a genome assembly using mapped paired end reads, 
+without the use of a reference genome for comparison."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['ftp://ftp.sanger.ac.uk/pub/resources/software/reapr/']
+sources = ['Reapr_%(version)s.tar.gz']
+patches = ['Reapr-1.0.18-cleanup.patch']
+checksums = [
+    '6d691b5b49c58aef332e771d339e32097a7696e9c68bd8f16808b46d648b6660',  # Reapr_1.0.18.tar.gz
+    '87131b45635815971e6e583712008043c3d077205127b50fbb09b9b5289b3b20'   # Reapr-1.0.18-cleanup.patch
+]
+
+dependencies = [
+    ('Perl', '5.28.1'),
+    ('R', '3.6.0'),
+    ('BamTools', '2.5.1'),
+    ('SAMtools', '1.9'),
+]
+
+builddependencies = [('CMake', '3.13.3')]
+
+exts_defaultclass = 'PerlModule'
+
+exts_list = [
+    ('File::Spec::Link', '0.140', {
+        'source_tmpl': 'File-Copy-Link-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/R/RM/RMBARKER/'],
+        'checksums': ['2063656dcd38bade43dc7f1e2ef5f1b6a8086c2f15d37b334189bd2a28e8ffeb'],
+    })
+]
+
+files_to_copy = [(['bin/*'], "bin")]
+
+sanity_check_paths = {
+    'files': ['bin/reapr'],
+    'dirs': ['lib/perl5'],
+}
```

Comment:
Is `reapr --version` is viable sanity check command (does it exit with `0`)?

If so, please add:

```python
sanity_check_commands = ["reapr --version"]
```

(`--help` is another common alternative)

Reply:
Typing just "reapr" gives the help page:
`REAPR version: 1.0.18
Usage:
    reapr <task> [options]
...
`
but unfortunately it ends with error code 255. No options like "--version" or "-V" are defined.

Reply:
In that case, this is an acceptable approach:

```python
sanity_check_commands = ["reapr 2>&1 | grep 'REAPR version: %(version)s Usage'"]
```

(the `2>&1` is required because `reapr` produces output to stderr)

Reply:
I've added this myself to this easyconfig via 0508e03